mapping.csv

```diff
@@ -234,7 +239,9 @@
 "^(b106-)?bexelets$",bexelets,used by microsoft
 "^(b106-)?bezigate$",bezigate,used by microsoft
 "^(b106-)?binder$",binder,used by microsoft
+"^(b106-)?bioazih$",bioazih,used by microsoft
 "^(b106-)?bisar$",bisar,used by microsoft
+"^(bisonal|korlia)$",bisonal,https://malpedia.caad.fkie.fraunhofer.de/details/win.korlia
```

Comment:
If malpedia maps bisonal to korlia, I'd prefer to simply follow malpedia here and use korlia as family

Reply:
What do you think about that?